pkg/vpa/vpa.go

```diff
@@ -118,7 +125,7 @@ func (r Reconciler) checkDeploymentLabels(deployment *appsv1.Deployment) (bool,
 	if len(deployment.ObjectMeta.Labels) > 0 {
 		for k, v := range deployment.ObjectMeta.Labels {
 			klog.V(7).Infof("Deployment Label - %s: %s", k, v)
-			if strings.ToLower(k) == "goldilocks.fairwinds.com/enabled" {
+			if strings.ToLower(k) == vpaEnabledLabel {
```

Comment:
We should consider switching these to use [ParseBool](https://golang.org/pkg/strconv/#ParseBool) instead so that more options will work.

Reply:
Oh yeah, that's a great idea! I'll go ahead and add this in a follow-up commit.

- [ ] Use `strcon.ParseBool` to verify boolean settings

Reply:
I also updated `namespaceIsManaged` to use `ParseBool` too